DBScript.sql

```diff
@@ -73,14 +73,18 @@ CREATE TABLE Address (
 
 CREATE TABLE Driver (
   id_Driver SERIAL PRIMARY KEY UNIQUE,
-  id_Address INTEGER NOT NULL REFERENCES Address(id_Address),
   first_name VARCHAR(30) NOT NULL CHECK(char_length(first_name)>2),
   last_name VARCHAR(30) NOT NULL CHECK(char_length(last_name)>2),
   pesel CHAR(11) UNIQUE NOT NULL CHECK(char_length(pesel)=11),
   salary DECIMAL(8,2) NULL DEFAULT 1600 CHECK(salary>=0),
   salary_bonus DECIMAL(8,2) DEFAULT 0,
   available boolean DEFAULT '1',
-  deleted boolean DEFAULT '0'
+  deleted boolean DEFAULT '0',
+  house_number VARCHAR(8) NOT NULL,
+  street VARCHAR(40) NOT NULL CHECK(char_length(street)>2),
+  town VARCHAR(30) NOT NULL CHECK(char_length(town)>1),
+  country VARCHAR(30) NOT NULL CHECK(char_length(country)>2),
+  code CHAR(10) NOT NULL
```

Comment:
IMO this approach is wrong - we break second normal form here (https://en.wikipedia.org/wiki/Second_normal_form). All other entities refer to Address entity, why Driver should have embedded Address fields?